.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -138,7 +138,8 @@ phases:
           SKU_NAME="1804gen2"; else \
           OS_DISK_SAS="$(cat packer-output | grep "OSDiskUriReadOnlySas:" | cut -d " " -f 2)"; \
           VHD_NAME="$(echo $OS_DISK_SAS | cut -d "/" -f 8 | cut -d "?" -f 1)"; \
-          SKU_NAME=$UBUNTU_SKU; fi &&
+          SKU_NAME=$UBUNTU_SKU; fi && \
+          if [[ "${AZURE_VM_SIZE}" == *"Standard_NC"* ]]; then SKU_NAME="${SKU_NAME}gpu"; fi && \
           docker run --rm \
```

Comment:
i don't get it, what is this SKU_NAME for?

Reply:
ok, so this becomes SIG definition name. It seems we should use the "fullgpu" feature flags to tell if it's for GPU or not.
Also we need to have gen1 or gen2 in the sku name

Reply:
update to use feature flag. gen2 was added at L138